src/stk/image/volume.h

```diff
@@ -227,19 +234,24 @@ namespace stk
         // Fills the volume with the specified value
         void fill(const T& value);
 
-        // Returns value at 
+        // Returns value at
         T at(int x, int y, int z, BorderMode border_mode) const;
         T at(int3 p, BorderMode border_mode) const;
 
-        T linear_at(float x, float y, float z, BorderMode border_mode) const;
-        T linear_at(float3 p, BorderMode border_mode) const;
+        // Indices in the voxel grid
+        T linear_at_index(float x, float y, float z, BorderMode border_mode) const;
+        T linear_at_index(float3 p, BorderMode border_mode) const;
+
+        // Point coordinates in image space
+        T linear_at_point(float x, float y, float z, BorderMode border_mode) const;
+        T linear_at_point(float3 p, BorderMode border_mode) const;
```

Comment:
Still for clarity, I added a method to interpolate from a point in image space.

Reply:
I think we should probably keep `linear_at` and then add something like `linear_at_world`.

Reply:
Ok, it is probably better to stick to the initial naming, so I am fixing it accordingly.

I was going for that names in part to be consistent with ITK (index vs point), and in part because I find less likely to get myself confused if both names are fully qualified.

Reply:
For me, `index` and `point` seems fine, `world` was merely an example. The thing I was trying to express was that I want to keep the regular `linear_at` name for index sampling. Mostly for selfish reasons because I'm a index type of guy. I think doing things in world coordinates should be an exception rather than a rule. Mostly because it complicates things. So having to write `linear_at_index` every time is a bit annoying.

Reply:
Haha, thank you. I see the point of using `linear_at_index`, but it still annoys me, because you don't write `vol.at_index(p)` 😄.